File: article-api/src/main/java/xyz/eastack/article/interfaces/AboutController.java
package xyz.eastack.article.interfaces;

import lombok.RequiredArgsConstructor;
import org.springframework.boot.info.BuildProperties;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import reactor.core.publisher.Mono;

@RestController
@RequestMapping("/about")
@RequiredArgsConstructor
public class AboutController {
    private final BuildProperties buildProperties;

    @GetMapping
    public Mono<BuildProperties> about() {
        return Mono.just(buildProperties);
    }
}

File: post-service-api/src/main/kotlin/me/eastack/post/about/AboutController.kt
package me.eastack.post.about

import me.eastack.post.sdk.AboutRepresentation
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(value = ["/about"])
class AboutController {

    @GetMapping
    fun about():AboutRepresentation {
        return AboutRepresentation(
            "post",
            "123",
            "2021-03-18 16:58",
            "2021-03-18 17:00",
            "acf671a",
            "main",
            "dev",
        )
    }
}